workflow/rules/commons/01_constants.smk

```diff
@@ -27,3 +27,6 @@ DIR_REPO = DIR_REPOSITORY
 
 DIR_WORKING = pathlib.Path(workflow.workdir_init).resolve(strict=True)
 WORKDIR = DIR_WORKING
+
+DIR_REFCON = pathlib.Path(config['reference_container_folder'])
```

Comment:
these options must take the possibility into account that these option have not been set in the config

Reply:
I hope I addressed that issue appropriately in the latest commit